tests/test_SklearnAdaBoostConverter.py

```diff
@@ -41,14 +42,31 @@ def test_ada_boost_classifier_samme(self):
         model.fit(X_train, y_train)
         model_onnx = convert_sklearn(model, 'AdaBoost classification',
                                      [('input',
-                                      FloatTensorType(X_train.shape))])
+                                      FloatTensorType(X_test.shape))])
         self.assertIsNotNone(model_onnx)
         dump_data_and_model(X_test.astype('float32'), model, model_onnx,
                             basename="SklearnAdaBoostClassifierSAMME",
                             allow_failure="StrictVersion("
                                           "onnxruntime.__version__)"
                                           "<= StrictVersion('0.2.1')")
 
+    @unittest.skip(reason="CumSum op has not yet been implemented in onnx")
```

Comment:
Do you know if there is anyone is working on this onnx Op? If not, we should not enable this kind of converter which can't run with any onnx-runtime.

Reply:
I created this task: https://aiinfra.visualstudio.com/Lotus/_workitems/edit/2740. It is still in unassigned state.